datahub-gms-graphql-service/src/main/java/com/linkedin/datahub/graphql/service/GmsGraphQLProvider.java

```diff
@@ -0,0 +1,34 @@
+package com.linkedin.datahub.graphql.service;
+
+import javax.annotation.PostConstruct;
+
+import org.springframework.context.annotation.Bean;
+import org.springframework.stereotype.Component;
+
+import com.linkedin.datahub.graphql.GmsGraphQLEngine;
+import com.linkedin.datahub.graphql.GraphQLEngine;
+
+import graphql.GraphQL;
+
+@Component
+public class GmsGraphQLProvider {
+
+    private GraphQL graphQL;
+    private GraphQLEngine graphQLEngine;
+
+    @Bean
+    public GraphQL graphQL() {
```

Comment:
This can be removed now, as it is unused, right?

Reply:
As shown here - https://www.graphql-java.com/tutorials/getting-started-with-spring-boot/#schema
`graphQL` bean is needed by the `graphql-java-spring-boot-starter-webmvc` dependency.